setup.py

```diff
@@ -67,7 +67,7 @@ def run(self):  # noqa: D102
     ],
     keywords="pyramid openapi3 openapi rest restful",
     packages=find_packages(exclude=["tests"]),
-    package_data={"pyramid_openapi3": ["static/*.*"], "": ["LICENSE"]},
+    include_package_data=True,
```

Comment:
We have `graft pyramid_openapi3` in MANIFEST.IN, which means "include all files in given directory", so this should be enough.